clients/src/test/java/org/apache/kafka/clients/consumer/KafkaConsumerTest.java

```diff
@@ -70,7 +72,6 @@
 
 public class KafkaConsumerTest {
 
-    private final String topic = "test";
```

Comment:
I think you did this change to trigger Jenkins. I'd probably revert it. If Jenkins has a transient failure, it's OK to leave it and let the reviewer run the tests locally.